packages/components/src/InputValidation/InputValidation.tsx

```diff
@@ -9,15 +9,21 @@ interface InputValidationProps {
    * Validation message to be displayed
    */
   readonly message: string;
+  readonly visible?: boolean;
 }
 
-export function InputValidation({ message }: InputValidationProps) {
+export function InputValidation({
+  message,
+  visible = true,
```

Comment:
Not a breaking change. If people use `InputValidation` it will render as before but they can now toggle with visible to use it in a new way. See: https://github.com/GetJobber/atlantis/pull/2184/files#diff-2dd45bcb75b2965ffe0b5fceb22778aaea0eb9af7c9912de07dae06ff3f7bafeR122-R127

Reply:
Should we update the storybook examples, add some tests? (I'm open to that being a followup PR! Let's just not forget, lol.)

Reply:
I can push some tests and update the storybook example. The commit should be very small